Dec2022Demo/LibForDec2022Demo.jl

```diff
@@ -0,0 +1,28 @@
+
+using AlgebraicPetri: vectorfield
+
+# AlgebraicDynamics: ContinuousResourceSharer
+
+valueat(x::Number, u, t) = x
+valueat(f::Function, u, t) = try f(u,t) catch e f(t) end
+AlgebraicPetri.vectorfield(pn::AbstractPetriNet) = begin
```

Comment:
Why are we overriding this? To add support for parameters that vary in time? I think there's already support for that in AlgebraicPetri.